packages/http-client/src/remote-admin-api.ts

```diff
@@ -82,23 +83,16 @@ export class RemoteAdminApi implements AdminApi {
   }
 
   async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
-    const code = await this.generateCode()
-    const body = modelIDsAsRequestBody(modelsIDs)
-    await this._fetchJson(this.getModelsUrl(), {
-      method: 'post',
-      body: {
-        jws: await this.buildJWS(this._getDidFn(), code, this.getModelsUrl().pathname, body),
-      },
-    })
+    await this.startIndexingModelData(convertModelIdsToModelData(modelsIDs))
```

Comment:
this worries me a bit.  Now if someone updates their client before updating their node, things will break.  We've never defined a standard upgrade order between the client and server, so far we've always allowed devs to upgrade in either order and maintained backwards compatibility in both directions.  I don't think that approach is sustainable long term - I think we will eventually (ideally soon) need to standardize and document whether it is expected that devs update their client or their server first / more often.  But until we set that expectation with our community, this change feels risky to me.

CC @oed

Reply:
We can just change to use the existing method, and leave the proxying to the daemon.